scripts/config.py

```diff
@@ -235,11 +235,17 @@ def full_adapter(name, active, section):
 
 # The baremetal configuration excludes options that require a library or
 # operating system feature that is typically not present on bare metal
-# systems. Features that are excluded from "full" won't be in "baremetal"
-# either (unless explicitly turned on in baremetal_adapter) so they don't
-# need to be repeated here.
+# systems.
+# It also excludes debugging features that increase the code size of other
+# modules sice they would typically be disabled in production in environments
```

Comment:
```suggestion
# modules since they would typically be disabled in production in environments
```

Reply:
I amended the last commit to fix the typo.